Sources/SwiftCliCore/NestedArrayBuilder.swift

```diff
@@ -3,26 +3,28 @@ import Foundation
 public class NestedArrayBuilder {
     public init() {}
 
-    public func arrayMaker(row: String) throws -> [String] {
+    public func arrayMaker(row: String) -> [String] {
         let substringRowStitches = row.split(separator: " ")
         let rowStitches = substringRowStitches.map {(String($0))}
-        let expandedRowStitches = try handleRepeats(row: rowStitches)
-        return(expandedRowStitches)
+        // let expandedRowStitches = try handleRepeats(row: rowStitches)
+        return(rowStitches)
     }
 
-    public func expandRow(row: [String]) throws ->  [String] {
-        return try row.flatMap { try getMultipleStitchInfo(stitch: $0) }
+    public func expandRow(row: [String]) -> [String] {
+        var rowWithExpandedMultipleStitches: [String] = []
+        do { try rowWithExpandedMultipleStitches = row.flatMap { try getMultipleStitchInfo(stitch: $0) } } catch { return row }
+        do { return try handleRepeats(row: rowWithExpandedMultipleStitches) } catch { return rowWithExpandedMultipleStitches }
     }
 
     private func handleRepeats(row: [String]) throws -> [String] {
 
         var repeatedRow: [String] = []
         var currentRowSection: [String] = []
-        for (index, stitch) in row.enumerated() {
-            if let repeats = (stitch.range(of: "^[(0-9x)]*$", options: .regularExpression)) {
+        for (_, stitch) in row.enumerated() {
```

Comment:
If you're not using the index anymore, then why do you still need `enumerated()`?